packages/domains-table/src/domains-table/domains-table-row-actions.tsx

```diff
@@ -35,6 +35,7 @@ interface DomainsTableRowActionsProps {
 	canSetPrimaryDomainForSite: boolean;
 	isSiteOnFreePlan: boolean;
 	isSimpleSite: boolean;
+	isHostingOverview?: boolean;
```

Comment:
Nothing for this specific PR, but does this variable refer to the same thing as the `inSiteContext`? If that is the case, I think we should converge to use the same name. `isHostingOverview` seems more readable to me.

Reply:
Yeah, that refers to the `hosting-overview` context, which is indeed a site context.